provisioning/solr.md

```diff
@@ -4,18 +4,7 @@ For ArchivesSpace > 3.1.1 this is **required**. For previous versions it is opti
 
 ## Supported Solr Versions
 
-ArchivesSpace is currently tested for compatibility with Solr version: `9 (9.4.1)`.
-This version is being used during development and the ArchivesSpace test suite passes
-with this version therefore is it recommended that you use this version of Solr in
-production.
-
-It may be possible to use ArchivesSpace with an older version of Solr. However in that case it
-is important to check the [release notes](https://github.com/archivesspace/archivesspace/releases)
-for any potential version compatibility issues.
-
-**Note: the ArchivesSpace Program Team can only provide support for Solr deployments
-using the "officially" supported version with the standard configuration provided by
-the application. Everything else will be treated as "best effort" community-led support.**
+See the [Solr requirement notes](../administration/getting_started.html)
```

Comment:
I would add the Solr hash to this link `(../administration/getting_started.html#solr)`

Reply:
The change has been committed